bazel/install-deps.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+set -e
+
+echo "installing redpanda toolchain"
+
+if [[ $EUID -ne 0 ]]; then
+  echo "This script should be run as root."
+  exit 1
+fi
+if [ -f "/etc/os-release" ]; then
+  . /etc/os-release
+elif [ -f "/etc/arch-release" ]; then
+  export ID=arch
+else
+  echo "/etc/os-release missing."
+  exit 1
+fi
+
+deb_deps=(
```

Comment:
For these is where `rules-nixpkg` could be handy in terms of determinism.

Reply:
ah right on. is anyone using these?

Reply:
We use `devenv` in Cloud to stand up the entire stack in our laptops. DevProd is using `devbox`.

Reply:
TBH it's better to move all tools into Bazel - I don't think that's a huge lift for this list. nixpkg is a fair amount of complexity, and I've had problems with clang toolchains and such not having a good development environment because of all the extra flags they inject to make things hermetic.